### DISCUSSION TITLE
引入configProvider报错无css文件

### DISCUSSION THREAD

**Discussion by grandzz** [ant-design/ant-design-mobile#5853] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/58963555/205226588-78c67b6f-1ccf-4b8a-a298-bca57dbe805c.png)
Internal server error: Failed to resolve import "/node_modules/antd-mobile/es/components/set-default-config/set-default-config.css" from "src\index.jsx". Does the file exist?

**Comment by miracles1919** [comment, marked as answer]
产物里是没有这个 css 的

**Reply by grandzz** [reply]
使用单独引入解决了这个问题
import { setDefaultConfig } from 'antd-mobile/es/components/config-provider'
